great_expectations/rule_based_profiler/types/data_assistant_result/data_assistant_result.py

```diff
@@ -199,18 +210,35 @@ def get_expect_values_to_be_between_chart(
         Returns:
             An altair line chart with confidence intervals corresponding to "between" expectations
         """
-        line_color: alt.HexColor = alt.HexColor(ColorPalettes.HEATMAP_6.value[4])
+        metric_plot_util: MetricPlotUtil = MetricPlotUtil(metric_name, metric_type)
+        domain_plot_util: DomainPlotUtil = DomainPlotUtil(domain_name, domain_type)
+        batch_id_plot_util: BatchIdPlotUtil = BatchIdPlotUtil(
+            "batch_id", AltairDataTypes.NOMINAL.value
+        )
+        return DataAssistantResult._get_expect_values_to_be_between_chart(
+            df=df,
+            metric_plot_util=metric_plot_util,
+            domain_plot_util=domain_plot_util,
+            batch_id_plot_util=batch_id_plot_util,
+            subtitle=subtitle,
+        )
 
-        metric_title: str = metric_name.replace("_", " ").title()
-        domain_title: str = domain_name.title()
+    @staticmethod
+    def _get_expect_values_to_be_between_chart(
+        df: pd.DataFrame,
+        metric_plot_util: MetricPlotUtil,
+        domain_plot_util: DomainPlotUtil,
+        batch_id_plot_util: BatchIdPlotUtil,
+        subtitle: Optional[str],
+    ) -> alt.Chart:
+        line_color: alt.HexColor = alt.HexColor(ColorPalettes.HEATMAP_6.value[4])
 
-        title: Union[str, alt.TitleParams] = f"{metric_title} per {domain_title}"
-        if subtitle:
-            title = alt.TitleParams(title, subtitle=[subtitle])
+        title: alt.TitleParams = DataAssistantResult._determine_plot_title(
+            metric_title=metric_plot_util.title,
+            domain_title=domain_plot_util.title,
+            subtitle=subtitle,
+        )
 
-        batch_id: str = "batch_id"
```

Comment:
If we like the pattern used herein, I'm happy to apply it to `min_value` and `max_value` in this PR.